src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -80,20 +80,28 @@ public partial class HttpClientHandler : System.Net.Http.HttpMessageHandler
         public HttpClientHandler() { }
         public bool AllowAutoRedirect { get { throw null; } set { } }
         public System.Net.DecompressionMethods AutomaticDecompression { get { throw null; } set { } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
I believe this is because they are not present in 4.6.1 -- see explanation in this commit
https://github.com/dotnet/standard/commit/622e5b391bff432242b0f6e94a840b2d8d7ae298 

Seems to me though that this attribute should exist in the reference assemblies for netstandard.dll, but not for .NET Core itself. I'm not sure how those are distinguished in code though. @weshaggard can speak to this, but he's out until next week: I wouldn't wait.